s2p/__init__.py

```diff
@@ -308,10 +308,31 @@ def disparity_to_ply(tile):
     xyz_array, err = triangulation.disp_to_xyz(rpc1, rpc2,
                                                np.loadtxt(H_ref), np.loadtxt(H_sec),
                                                disp_img, mask_rect_img,
-                                               int(cfg['utm_zone'][:-1]),
                                                img_bbx=(x, x+w, y, y+h),
                                                A=np.loadtxt(pointing))
 
+    # reshape the xyz array into a 3-column 2D-array
+    xyz_shape = xyz_array.shape
+    xyz_list = xyz_array.reshape(-1, 3)
+
+    # output EPSG conversion
+    in_epsg = 4979
+    if 'out_epsg' in cfg:
```

Comment:
I would rather have a block like:
```python
if cfg['out_epsg'] is None:
    cfg['out_epsg'] = the local utm zone's epsg
```
somewhere in `s2p.initialisation`, and then consider everywhere downstream in the code that `cfg['out_epsg']` is defined.

What do you think about that?

Reply:
Also related, I would prefer to have a uniform `proj_com` in the ply files generated, rather than keeping producing some ply files with `UTM ...` and others with `EPSG ...`.
And `cfg['utm_zone']` should be dropped from everywhere in the code if possible to avoid confusion/silent inconsistencies